packages/moxb/src/globals.d.ts

```diff
@@ -1 +1,3 @@
 declare module 'pretty';
+
+declare module 'btoa-lite';
```

Comment:
Is this okay, just to declare an empty module? What is `@types/atob ` is used for?

Reply:
This basically means "any".
Unfortunately I found no ready-made typedefs, and I didn't find it to be important enough to warrant creating one.